### PR TITLE
Fix for read property '1' of undefined

### PR DESCRIPTION
**Pull request by Dillon-Dumesnil**

### CODE REVIEW

File: src/components/CreateCourseRunPage/index.jsx [hunk 1]
```diff
@@ -138,7 +138,8 @@ class CreateCourseRunPage extends React.Component {
     const parsedTypeOptions = courseOptionsData && courseOptionsData.type &&
       parseCourseTypeOptions(courseOptionsData.type.type_options);
     const courseRunTypeOptions = parsedTypeOptions && parsedTypeOptions.courseRunTypeOptions;
-    const defaultRunType = courseRunTypeOptions && courseRunTypeOptions[type][1].value;
+    const defaultRunType = courseRunTypeOptions && courseRunTypeOptions[type] &&
+      courseRunTypeOptions[type][1].value;
```
Comment (mikix) [inline src/components/CreateCourseRunPage/index.jsx]: Do you also want to add a `&& courseRunTypeOptions[type][1]`?

Comment (Dillon-Dumesnil) [reply src/components/CreateCourseRunPage/index.jsx]: Can't hurt. Going to update test too

File: src/components/CreateCourseRunPage/index.jsx [hunk 2]
```diff
@@ -138,7 +138,8 @@ class CreateCourseRunPage extends React.Component {
...
```
Comment (xnick421) [inline src/components/CreateCourseRunPage/index.jsx]: defaultRunType will be `undefined` if there are not atleast 2 elements in the `courseRunTypeOptions[type]` array.  is this expected?

Comment (Dillon-Dumesnil) [reply src/components/CreateCourseRunPage/index.jsx]: Yeah, no problems with that. Just sets the initialValues array and it being undefined there will just leave it as forcing them to select an option.

Comment (xnick421) [reply src/components/CreateCourseRunPage/index.jsx]: great

**Pull request merged**